src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,514 @@
+                "aeo_electricity_sector_fuel_type",
+                "net_summer_capacity_mw",  # should this be capacity_mw_eia?
+                # should we include "first modeled year" as a field, to help interpret all the cumulative fields?
+                "net_summer_capacity_cumulative_planned_additions_mw",
+                "net_summer_capacity_cumulative_unplanned_additions_mw",
+                "net_summer_capacity_cumulative_total_additions_mw",  # this is just planned + unplanned - do we even care about publishing this?
+                "net_summer_capacity_cumulative_retirements_mw",
+                "gross_generation_mwh",  # chose gross bc the numbers add up to "total generation" which then equals "sales to customers + generation for own use" - but... is that right?
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_electricity_sector_fuel_type",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__generation_end_use_sector": {
+        "description": (
+            "Projected generation capacity & total generation in the end-use "
+            "sector.\n"
+            "Includes combined-heat-and-power plants and electricity-only "
+            "plants in the commercial and industrial sectors; and small on-site "
+            "generating systems in the residential, commercial, and industrial "
+            "sectors used primarily for own-use generation, but which may also "
+            "sell some power to the grid. Solar photovoltaic capacity portion of "
+            "Renewable Sources in gigawatts direct current; other technologies "
+            "in gigawatts alternating current."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_end_use_sector_fuel_type",
+                "net_summer_capacity_mw",  # should this be capacity_mw_eia?
+                "gross_generation_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_end_use_sector_fuel_type",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__electricity_sales": {
+        "description": "Projected electricity sales.",
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_sales_sector",
```

Comment:
We could unstack this since there's only residential/industrial/commercial/transportation/total - should we?